scripts/migrate.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+(async () => {
```

Comment:
This requires Node.js 8, doesn't it? Should be documented at least in package.json, also in the Readme.

Reply:
Documented in `package.json` and Readme. :)